cheroot/ssl/pyopenssl.py

```diff
@@ -216,6 +216,8 @@ def __new__(mcl, name, bases, nmspc):
             'settimeout',
             'gettimeout',
             'shutdown',
+            'recv_into',
+            '_decref_socketios',
```

Comment:
Why are these needed?

Reply:
Okay, with the plan to remove the proxy in the future, it's probably not worth it trying to understand it deeper. Marking as resolved.